### PR TITLE
tests: re-enable kernel-module-load tests on arm

### PR DESCRIPTION
**Pull request by mardy**
Find another pair of modules which we can use on all tested devices.

The problem with the previous modules was that on the PI the pcspkr module was missing, and the parport_pc was causing a (recoverable) kernel crash.

### CODE REVIEW

File: tests/main/interfaces-kernel-module-load/test-snapd-kernel-module-load/meta/snap.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ plugs:
...
               load: denied
-            - name: parport_pc
+            - name: bfq
               load: on-boot
-              options: io=0x3bc,0x278 irq=none
+              options: slice_idle_us=20 strict_guarantees=1
             - name: other_module
               load: denied
-            - name: pcspkr
+            - name: arc4
```
Comment (stolowski) [inline tests/main/interfaces-kernel-module-load/test-snapd-kernel-module-load/meta/snap.yaml:16]: Why these changes, is this because these modules are common for all platforms?

Comment (mvo5) [reply tests/main/interfaces-kernel-module-load/test-snapd-kernel-module-load/meta/snap.yaml:16]: Yeah, maybe worth to extend the commit message here a bit so that it includes information that one of the current modules crashes the kernel and the other is not available?

Comment (mardy) [reply tests/main/interfaces-kernel-module-load/test-snapd-kernel-module-load/meta/snap.yaml:16]: I hope they are :-) The problem was that on the PI the `pcspkr` module was missing, and the `parport_pc` was causing a (recoverable) kernel crash.

**Review by stolowski — APPROVED** [review]
LGTM, but please add a short comment explaining the choice of kernel modules. Thanks!

**Review by mvo5 — APPROVED** [review]
Thank you

**Comment by mardy** [comment]
Here's the test results on a RPI3:
```
2022-01-11 14:41:02 Sending project content to external:ubuntu-core-16-arm-32...
2022-01-11 14:41:05 Preparing external:ubuntu-core-16-arm-32 (external:ubuntu-core-16-arm-32)...
2022-01-11 14:41:06 Preparing external:ubuntu-core-16-arm-32:tests/main/ (external:ubuntu-core-16-arm-32)...
2022-01-11 14:42:28 Preparing external:ubuntu-core-16-arm-32:tests/main/interfaces-kernel-module-load (external:ubuntu-core-16-arm-32)...
2022-01-11 14:42:43 Executing external:ubuntu-core-16-arm-32:tests/main/interfaces-kernel-module-load (external:ubuntu-core-16-arm-32) (1/1)...
2022-01-11 14:42:50 Restoring external:ubuntu-core-16-arm-32:tests/main/interfaces-kernel-module-load (external:ubuntu-core-16-arm-32)...
2022-01-11 14:42:57 Restoring external:ubuntu-core-16-arm-32:tests/main/ (external:ubuntu-core-16-arm-32)...

2022-01-11 14:42:57 Restoring external:ubuntu-core-16-arm-32 (external:ubuntu-core-16-arm-32)...
2022-01-11 14:42:58 Discarding external:ubuntu-core-16-arm-32...
2022-01-11 14:42:58 Successful tasks: 1
2022-01-11 14:42:58 Aborted tasks: 0
```
